releases.md

```diff
@@ -277,10 +294,94 @@ For more information on testing queued jobs, check out the [queue documentation]
+2024_03_01_123456_create_telescope_entries_table.php // [tl! add]
+```
+
+For more information on migrations, check out the [migrations documentation](/docs/{{version}}/migrations).
+
+### Resend Mail Driver
+
+_Resend mail driver was contributed by [Taylor Otwell](https://github.com/taylorotwell)_.
+
+Laravel 11 includes a new `resend` mail driver that allows you to use the [Resend](https://resend.com) mail service to send your application's emails.
+
+For more information on sending mails in Laravel, check out the [mail documentation](/docs/{{version}}/mail).
+
+### Model Casts Improvements
+
+_Model casts improvements were contributed by [Nuno Maduro](https://github.com/nunomaduro)_.
+
+Laravel 11 supports defining your model's casts using a method instead of a property. This allows for a much cleaner way to define casts for your models, especially when using casts with arguments:
+
+    /**
+     * Get the attributes that should be cast.
+     *
+     * @return array<string, string>
+     */
+    protected function casts(): array
+    {
+        return [
+            'options' => AsCollection::using(OptionCollection::class),
+                      // AsEncryptedCollection::using(OptionCollection::class),
+                      // AsEnumArrayObject::using(OptionEnum::class),
+                      // AsEnumCollection::using(OptionEnum::class),
+        ];
+    }
+
+For more information on attribute casting, check out the [eloquent mutators documentation](/docs/{{version}}/eloquent-mutators#attribute-casting).
+
+### `once` helper
+
+_The `once` helper was contributed by [Taylor Otwell](https://github.com/taylorotwell)_ and _[Nuno Maduro](https://github.com/nunomaduro)_.
+
+Laravel 11 includes a new `once` helper that is extremely useful for caching a costly computation without the need to create a temporary like a nullable property on the class itself. Here is a practical example:
+
+```php
+class User extends Model
+{
+    public function stats(): array
+    {
+        return once(function () {
+            // Any kind of expensive computation that should be cached during the request...
+
+            return $stats;
+        });
+    }
+}
+```
+
+For more information on the `once` helper, check out the [helpers documentation](/docs/{{version}}/helpers#method-once).
+
+### Improved Performance When Testing With In-Memory Databases
+
+_Improved performance when testing with in-memory databases was contributed by [Anders Jenbo](https://github.com/AJenbo)_
+
+When using the `:memory:` SQLite database for testing, Laravel 11 tests will experience improved speed. This is due to the fact that Laravel now maintains a reference to PHP's PDO object and reuses it across connections. This results in a significant performance boost, with test suites that used to take minutes to run now being completed in just seconds.
+
+### Improved Support For MariaDB
+
+_Improved support for MariaDB was contributed by [Julius Kiekbusch](https://github.com/Jubeki)_
```

Comment:
```suggestion
_Improved support for MariaDB was contributed by [Jonas Staudenmeir](https://github.com/staudenmeir) and [Julius Kiekbusch](https://github.com/Jubeki)_
```

Thanks! While I am honoured to be mentioned here, I only added the mariadb config entry and the installation option to the laravel installer, while  @staudenmeir contributed the driver itself.